wren-base/src/test/java/io/wren/base/sqlrewrite/analyzer/TestDecisionPointAnalyzer.java

```diff
@@ -185,6 +190,20 @@ public void testSelectItem()
         assertThat(result.size()).isEqualTo(1);
         assertThat(result.get(0).getSelectItems().size()).isEqualTo(1);
         assertThat(result.get(0).getSelectItems().get(0).getExpression()).isEqualTo("c.*");
+
+        statement = parseSql("SELECT customer_name FROM orders");
+        result = DecisionPointAnalyzer.analyze(statement, DEFAULT_SESSION_CONTEXT, mdl);
+        assertThat(result.size()).isEqualTo(1);
+        assertThat(result.get(0).getSelectItems().size()).isEqualTo(1);
+        assertThat(result.get(0).getSelectItems().get(0).getExpression()).isEqualTo("customer_name");
+        assertThat(result.get(0).getSelectItems().get(0).getExprSources()).isEqualTo(List.of(new ExprSource("customer_name", "orders", "customer_name", new NodeLocation(1, 8))));
```

Comment:
We won't dig into the expression of a calculated field. So, in the `exprSource`, we only provide itself.